dags/burnham.py

```diff
@@ -178,9 +180,9 @@
 """
 
 WANT_TEST_EXPERIMENTS = [
-    {"experiment": "no_experiments", "document_count": 2},
-    {"experiment": "spore_drive:tardigrade", "document_count": 2},
-    {"experiment": "spore_drive:tardigrade-dna", "document_count": 6},
+    {"experiment": "no_experiments", "document_count": 4},
+    {"experiment": "spore_drive:tardigrade", "document_count": 4},
```

Comment:
From what I can tell we should expect `1` additional `discovery` ping, see https://github.com/mozilla/telemetry-airflow/pull/1382#pullrequestreview-784554475.

```suggestion
    {"experiment": "spore_drive:tardigrade", "document_count": 3},
```

Reply:
Whoops, I had that but maybe it got lost when I had to do a rebase/merge. Just going to add it via github.